tests/dialects/test_snowflake.py

```diff
@@ -375,31 +375,31 @@ def test_snowflake(self):
         self.validate_all(
             "SELECT TO_TIMESTAMP(1659981729)",
             write={
-                "bigquery": "SELECT UNIX_TO_TIME(1659981729)",
+                "bigquery": "SELECT TIMESTAMP_SECONDS(1659981729)",
                 "snowflake": "SELECT TO_TIMESTAMP(1659981729)",
                 "spark": "SELECT CAST(FROM_UNIXTIME(1659981729) AS TIMESTAMP)",
             },
         )
         self.validate_all(
             "SELECT TO_TIMESTAMP(1659981729000, 3)",
             write={
-                "bigquery": "SELECT UNIX_TO_TIME(1659981729000, 'millis')",
+                "bigquery": "SELECT TIMESTAMP_MILLIS(1659981729000)",
                 "snowflake": "SELECT TO_TIMESTAMP(1659981729000, 3)",
                 "spark": "SELECT TIMESTAMP_MILLIS(1659981729000)",
             },
         )
         self.validate_all(
             "SELECT TO_TIMESTAMP('1659981729')",
             write={
-                "bigquery": "SELECT UNIX_TO_TIME('1659981729')",
+                "bigquery": "SELECT TIMESTAMP_SECONDS('1659981729')",
```

Comment:
This seems invalid BQ - we need this to be an int instead of a string apparently, see https://cloud.google.com/bigquery/docs/reference/standard-sql/timestamp_functions#timestamp_seconds.

Perhaps it makes sense to add a cast @ generation time to INT64 to be on the safe side?